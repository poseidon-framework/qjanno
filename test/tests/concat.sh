qhs -H -O "SELECT foo||baz FROM big.csv WHERE bar IS NOT NULL"
