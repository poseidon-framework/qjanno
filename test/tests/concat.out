foo||baz
a0a2
b1b1
a0a1
b0b2
a0a3
b1b3
a0a3
